sdks/python/setup.py

```diff
@@ -87,6 +87,7 @@ def get_version():
     'avro>=1.7.7,<2.0.0',
     'dill>=0.2.5,<0.3',
     'google-apitools>=0.5.2,<1.0.0',
+    'googledatastore==6.3.0',
```

Comment:
6.3.0 is the their latest. They could be backward compatible, but their dependencies like protobuf might now. I will like to us to carefully examine dependencies before we upgrade.